lib/perl/Genome/Qc/Tool/Picard/CalculateHsMetrics.t

```diff
@@ -53,19 +51,43 @@ my $alignment_result = Genome::Test::Factory::InstrumentData::AlignmentResult->s
     instrument_data => $instrument_data,
 );
 
-my $tool = $pkg->create(
-    gmt_params => {
-        bait_intervals => $temp_file,
-        input_file => $temp_file,
-        output_file => $output_file,
-        target_intervals => $temp_file,
-        temp_directory => $temp_file,
-        use_version => 1.123,
+my $bam_file = abs_path(File::Spec->join($data_dir, 'speedseq_merged.bam'));
+my $bait_intervals = abs_path(File::Spec->join($data_dir, 'bait.intervals'));
+my $target_intervals = abs_path(File::Spec->join($data_dir, 'target.intervals'));
```

Comment:
Good idea. This works for all tools except verifyBamId where it complains that it can't find the bam index. So I'll have to abs_path the whole bam file or symlink in the index file.